README.md

```diff
@@ -35,15 +35,45 @@ See [**Contributing.md**](./CONTRIBUTING.md) to see how to format your pull requ
 
 ## A to Z Order of Companies
 
-### A
+### A  
+
+### C  
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
+- **How to sign up**: 
+    - [Hacktoberfest Website](https://hacktoberfest.digitalocean.com)
+    - [Camunda website](https://camunda.com/hacktoberfest2021/)
+- **Notes**:  
+  - For your PR to count it must be:
+    - Submitted in a Camunda repo, AND
+    - The PR is labelled as ```hacktoberfest-accepted``` by a maintainer  
+  - Once your PR's are approved fill out the submission form [here](https://camunda.com/hacktoberfest2021/)  
+
+
 
 ## Least Involvement to Most Involvement
 
 ### Any - 1 PR
 
 ### 1 Merged PR
 
-### 2 or more PRs
+### 2 or more PRs  
+
+#### **Camunda**
+
+- **Swag**: T-shirt, stickers
+- **Requirements**: 4 pull requests to any Camunda repo
```

Comment:
Double check the order of things here. 😄 The A->Z and Least -> Most Involvement sections have a different order. Check the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/CONTRIBUTING.md) file and [last year's branch](https://github.com/crweiner/hacktoberfest-swag-list/tree/2020) for examples of how to set this up.

Reply:
Oh my bad, actually both prizes are for the same level of involvement. I should have written 'T-shirt and stickers'.